config/jest/setup.js

```diff
@@ -4,6 +4,7 @@ jest.unmock('promise');
 jest.unmock('whatwg-fetch');
 jest.unmock('object-assign');
 
+require('@babel/polyfill');
```

Comment:
Should use [(the original version) of `.storybook/polyfills.js`](https://github.com/carbon-design-system/carbon-components-react/blob/087604a/.storybook/polyfills.js) instead.

Reply:
The big issue is that we need `regeneratorRuntime` for `async` support, which is why this is being brought in.

Reply:
Have you tried `babel-plugin-transform-runtime` with `polyfill` option turned off? You can refer to how vanilla test env is set up.

Reply:
Definitely can use that instead!